test/integration/pledge_fulfillment_test.rb

```diff
@@ -51,7 +51,9 @@ class PledgeFulfillmentTest < ActionDispatch::IntegrationTest
     it 'should show a link to the pledge fulfillment tab after pledge sent' do
       find('#submit-pledge-button').click
       find('#pledge-next').click
+      assert has_text? 'Download your pledge here!'
```

Comment:
pretty sure this will cause a failing integration spec right now since you changed the copy? @colinxfleming would you want to keep this in or should we just remove this line?